tests/console/yast2_clone_system.pm

```diff
@@ -44,22 +44,19 @@ sub run {
 
     unless (is_opensuse) {
         my $devel_repo = get_required_var(is_sle('>=15') ? get_repo_var_name("MODULE_DEVELOPMENT_TOOLS") : 'REPO_SLE_SDK');
-        zypper_call "ar -c $utils::OPENQA_FTP_URL/" . $devel_repo . " devel-repo";
+        my $uri        = (is_staging) ? $devel_repo : "$utils::OPENQA_FTP_URL/" . $devel_repo;
+        zypper_call "ar -c $uri devel-repo";
     }
-
     zypper_call '--gpg-auto-import-keys ref';
-    zypper_call 'install jing';
 
-    my $rc_jing    = script_run "jing $xml_schema_path $ay_profile_path";
-    my $rc_xmllint = script_run "xmllint --noout --relaxng $xml_schema_path $ay_profile_path";
+    zypper_call 'install jing';
+    zypper_call "rr devel-repo" if (is_staging);
+    my $output_jing = script_output "jing $xml_schema_path $ay_profile_path";
 
-    if (($rc_jing) || ($rc_xmllint)) {
+    if ($output_jing) {
```

Comment:
Hmm, I'm not sure if I fully get it. Documentation states:
```
Executing script inside SUT with bash -eox (in case of serial console with bash -eo) and directs stdout (not' stderr!) to the serial console and returns the output if' the script exits with 0. Otherwise the test is set to failed. 
```

So for me it sound like you cannot do soft-failure here in case jing fails. Nevertheless, there is `proceed_on_failure` flag which we can reuse. TBH, I don't understand why you want to replace `script_run` with `script_output`.